openstef/pipeline/create_basecase_forecast.py

```diff
@@ -41,10 +42,10 @@ def create_basecase_forecast_pipeline(
     logger = structlog.get_logger(__name__)
 
     logger.info("Preprocessing data for basecase forecast")
-    # Validate and clean data - use a very long flatliner threshold.
-    # If a measurement was constant for a long period, a basecase should still be made.
+    # Validate data
+    # Currently effectively disabled by giving np.inf
     validated_data = validation.validate(
-        pj["id"], input_data, flatliner_threshold=4 * 24 * 14 + 1
+        pj["id"], input_data, flatliner_threshold=np.inf
```

Comment:
Just wondering why we don't remove the validate method? What are reasons to keep it with np.inf?

Reply:
I thought about that as well. I kept it in so we explicitly show that we did not 'forget' this step, and so that in the future we can more easily switch to arguments defined in the prediction_job (KTPS-1963)

Reply:
Ok good plan and good reasons :)